command_list.md

```diff
@@ -0,0 +1,116 @@
+# Account
+```
+rucio account list
+rucio account add
+rucio account show
+rucio account remove
+rucio account history
+rucio account update
+rucio account ban add
+rucio account ban remove
+rucio account attribute list
+rucio account attribute add
+rucio account attribute remove
+rucio account limit list
+rucio account limit add
+rucio account limit remove
+rucio account identity list
+rucio account identity add
+rucio account identity remove
+```
+
+# Config 
+```
+rucio config list
+rucio config add
+rucio config remove
+```
+
+# DID 
+```
+rucio did list
+rucio did show
+rucio did remove
+rucio did touch
```

Comment:
Do we want to keep `did touch`? This operation always felt like a hack. Maybe it can be put as port of `did update`.

Reply:
I wasn't sure why it was used to begin with so I left it as is? Maybe a better solution is to just have that be the argue-less behavior of `update`.

Reply:
It doesn’t even have a description. Very nice. I guess that at some point someone asked to be able to fake that a DID was used. I suggest you implement it at `did update --touch` for now and consider removing it entirely in a future version.